src/components/ConnectedTile.jsx

```diff
@@ -0,0 +1,45 @@
+import styles from '../styles/connectedTile'
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'cozy-ui/react/I18n'
+import { CONNECTION_STATUS } from '../lib/CollectStore'
+import { getAccountName } from '../lib/helpers'
+
+import { getConnectionStatusForTrigger } from '../ducks/connections'
+
+import KonnectorTile from './KonnectorTile'
+
+const ConnectedTile = props => <KonnectorTile {...props} />
```

Comment:
I don't understand the point of this, why can't you use `KonnectorTile` directly?

Reply:
`ConnectedTile` needs some "real time" information from the state to display the connection status (done, error, running, etc.). This is not the case for `KonnectorTile` as they are now just used to display a konnector.